libs/ngx-nuts-and-bolts/src/lib/pipes/enum-property/readme.md

```diff
@@ -0,0 +1,69 @@
+# Enum property pipe
+
+Common practice of working with enums often runs into a need to define separate metadata for each value of a given enum type. See [Infinum Handbook - Enumerations](https://infinum.com/handbook/frontend/angular/angular-guidelines-and-best-practices/file-and-module-organization-and-naming#enumerations)
+
+## 1. Features
+
+`enumProperty` pipe gives you an easy way to access given enum's value metadata in template.
+
+## 2. Usage
+
+Asssuming that you keep your enum definitions somewhat similar to what is defined in handbook like so:
```

Comment:
I would remove this sentence, let's assume that people are structuring the enums as described in the handbook. From the example, it will be obvious to them whether or not they are following the handbook

Reply:
b033fa5